src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -36,6 +36,21 @@
          [:span {:style {:position "absolute" :top "-50%" :fontSize "200%" :fontWeight "normal"}}
           "+"]])])})
 
+(react/defc UploadForm
+            {:render
+             (fn [{:keys []}]
+               [:div {}
+                [:form {:action "http://httpbin.org/post" :encType "multipart/form-data" :method "post"}
+                 [:div {:style {:padding "22px 48px 40px" :boxSizing "inherit" }}
+                  (style/create-form-label "Workspace Name")
+                  (style/create-text-field {:style {:width "33%"} :name "workspaceName"})
+                  (style/create-form-label "Workspace Namespace")
+                  (style/create-text-field {:style {:width "33%"} :name "workspaceNamespace"})[:br]
+                  [:input {:type "file" :name "entities"}]
+                  [:input {:type "submit" :value "Upload"}]]
+                 ]]
+               )})
```

Comment:
Nit: indentation for this function, and the close )]} should all go on the end of line 50.  Same in several other places.